packages/flutter_tools/lib/src/widget_preview/preview_details.dart

```diff
@@ -20,6 +22,12 @@ final class PreviewDetails {
   /// The file:// URI pointing to the script in which the preview is defined.
   final Uri scriptUri;
 
+  /// The line at which the Preview annotation was applied.
+  final int line;
```

Comment:
nit: in docs for line/char fields, can we include whether these are 1-based or 0-based. All of the tools and specs are not consistent so it's useful for the intentions to be documented if we need to track down off-by-one bugs (which have definitely not happened before 😄)

Reply:
Done!